src/pages/staff.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
```

Comment:
Let's rename the para folder to staff, and move this route to `/pages/staff/index.tsx`.